packages/lesswrong/server/notificationCallbacks.tests.ts

```diff
@@ -1,41 +1,112 @@
 import { testStartup } from "../testing/testMain";
-// import { createDummyPost } from "../testing/utils";
-// import { getSubscribedUsers, postsNewNotifications } from "./notificationCallbacks";
+import { createDummyPost, createDummyLocalgroup } from "../testing/utils";
+import { postsNewNotifications } from "./notificationCallbacks";
+import { createNotifications } from "./notificationCallbacksHelpers";
 
 testStartup()
 
-jest.mock('./notificationCallbacks', () => {
-  const originalModule = jest.requireActual('./notificationCallbacks');
+jest.mock('./notificationCallbacksHelpers', () => {
```

Comment:
In order to mock some functions to test `postsNewNotifications()`, I had to move them to a separate module